doc/releases.md

```diff
@@ -95,55 +118,46 @@ See https://github.com/nodejs/node/labels/confirmed-bug for complete and current
 
 ### Commits
 
-* Include the full list of commits since the last release here
+* Include the full list of commits since the last release here. Do not include "Working on X.Y.Z+1" commits.
 ```
 
-### 4. Update _src/node_version.h_
+The release type should be either Stable, LTS, or Maintenance, depending on the type of release being produced.
```

Comment:
@jasnell since the audience for this document is the release and LTS team members, and I'm not totally up to speed on the LTS release process, would you be OK with adding the LTS specific info in a subsequent PR?